src/pug/includes/body.pug

```diff
@@ -31,9 +31,12 @@ include ./fork.pug
                             option(value="DIFFICULTY_DESCENDING" class="difficulty") Difficulty, reversed
                     span order
 
-            .bottom
+            .bottom.flex
```

Comment:
We need to merge this into the previous row. It was acceptable (although not really okay) when there was only one option, but that's not the case anymore.

This PR should ref #92 as that's one of the things to do

Things to check

- [ ] wraps properly on smaller screens

Reply:
I've tried to fix this in 5e26d87ef0c5c7a3a2d1328183b98bd0f5f1d66b, but for some reason when building locally the wrapping didn't seem to work properly. I'm probably doing the Stylus selector and selector nesting wrong.
It would be great if you could take a look and let me know what you think should to be changed.

Reply:
Thanks for the reminder, this completely fell off my radar! The layout is 100% broken now, so I'm going to go through your commits and see which one broke it, then we can work from there 😇